analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -934,6 +1026,20 @@ def main(args):
     """
     logger.setup_logger(args.verbose if 'verbose' in args else None)
 
+    # Validate analyzer and checker config (if any)
+    config_validator = {
+        'analyzer_config': is_analyzer_config_valid,
+        'checker_config': is_checker_config_valid
+    }
+
+    config_validator_res = [validate_func(getattr(args, conf))
+                            for conf, validate_func in config_validator.items()
+                            if conf in args]
+
+    if False in config_validator_res \
+            and 'no_missing_checker_error' not in args:
+        sys.exit(1)
```

Comment:
If the validation failed and this suppression flag isn't present, try to advertise it, somewhat like what I've done in #3866:
```suggestion
        LOG.info("Although it is not reccomended, if you want to "
                 "suppress errors relating to unknown "
                 "analyzer/checker configs, consider using the option "
                 "'--no-missing-checker-error'")
        sys.exit(1)
```